progress2.inc

```diff
@@ -10,15 +10,15 @@
 #define _progress2_included
 
 #include <a_samp>
-#include <logger>
-#include <YSI_Data\y_iterate>
+#tryinclude <logger>
+#tryinclude <YSI_Data\y_iterate>
```

Comment:
This is not a [stacked PR](https://benjamincongdon.me/blog/2022/07/17/In-Praise-of-Stacked-PRs/) since it includes all parent commits.

When I said split the PR up, I meant separate the individual changes so each pull request is an isolated atomic set of changes.

You can do this with `git rebase -i master` and remove any commits not related to the actual feature you're proposing. Or `git cherry-pick xyz123` with the commit hash to copy a specific commit onto a new branch.